File: src/main/java/cc/moecraft/logger/environments/ConsoleEnv.java
package cc.moecraft.logger.environments;

import static cc.moecraft.logger.utils.AnsiUtils.removeFormat;

/**
 * 此类由 Hykilpikonna 在 2018/07/03 创建!
 * Created by Hykilpikonna on 2018/07/03!
 * Github: https://github.com/hykilpikonna
 * Meow!
 *
 * @author Hykilpikonna
 */
public class ConsoleEnv extends LogEnvironment
{
    @Override
    public void logRaw(String message)
    {
        System.out.println(removeFormat(message));
    }
}

File: src/main/kotlin/org/hydev/logger/environments/ConsoleEnv.kt
package org.hydev.logger.environments

import org.hydev.logger.withoutFormat

/**
 * 此类由 Hykilpikonna 在 2018/07/03 创建!
 * Created by Hykilpikonna on 2018/07/03!
 * Github: https://github.com/hykilpikonna
 * QQ: admin@moecraft.cc -OR- 871674895
 *
 * @author Hykilpikonna
 */
class ConsoleEnv: LogEnvironment()
{
    override fun logRaw(message: String)
    {
        println(message.withoutFormat())
    }
}
